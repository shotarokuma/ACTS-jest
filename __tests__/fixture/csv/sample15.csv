Column1	Column2	Column3	Column4	Column5	Column6	Column7	Column8	Column9	Column10	Column11	Column12	Column13	Column14	Column15	Column16	Column17	Column18	Column19	Column20	Column21	Column22	Column23	Column24	Column25	Column26	Column27	Column28	Column29	Column30	Column31	Column32	Column33	Column34	Column35	Column36	Column37	Column38	Column39	Column40	Column41	Column42	Column43	Column44	Column45	Column46	Column47	Column48	Column49	Column50
0	53	82	false	96	57	9	true	76	false	9	67	75	true	61	true	true	true	false	false	false	78	74	12	true	28	20	true	72	0	false	32	71	true	19	82	false	83	15	false	false	4	true	80	29	35	90	38	45	false	93	true	64	true	false	77	24	16	true	true