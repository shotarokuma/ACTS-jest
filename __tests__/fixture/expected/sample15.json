[
{"0":0,"1":53,"2":82,"3":false,"4":96,"5":57,"6":9,"7":true,"8":76,"9":false,"10":9,"11":67,"12":75,"13":true,"14":61,"15":true,"16":true,"17":true,"18":false,"19":false,"20":false,"21":78,"22":74,"23":12,"24":true,"25":28,"26":20,"27":true,"28":72,"29":0,"30":false,"31":32,"32":71,"33":true,"34":19,"35":82,"36":false,"37":83,"38":15,"39":false,"40":false,"41":4,"42":true,"43":80,"44":29,"45":35,"46":90,"47":38,"48":45,"49":false,"50":93,"51":true,"52":64,"53":true,"54":false,"55":77,"56":24,"57":16,"58":true,"59":true}
]
